aztec/src/main/kotlin/org/wordpress/aztec/AlignmentApproach.kt

```diff
@@ -0,0 +1,3 @@
+package org.wordpress.aztec
+
+enum class AlignmentApproach { SPAN_LEVEL, VIEW_LEVEL }
```

Comment:
One thing I'm not particularly fond of regarding the `VIEW_LEVEL` constant is that it has no meaning in the Aztec codebase other than _avoiding_ to use the IAztecAlignmentSpan-flavoured spans. I.e. there's no test that uses it, and no html serialization/deserialization that gets affected by it. At the end of the day, it feels that we just need a boolean to govern "let IAztecAlignmentSpan or not", and that's the whole operation.

So, what do you think about introducing a `avoidSpanAlignment` boolean instead of the enum?

Also, how about linking somewhere to the code that does utilize the view-level gravity to implement the alignment? I think that code is https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java#L325-L357?

Reply:
Added a comment to clarify this. With the comment, I'm not sure linking to the code adds much, but what do you think?

I also thought it might help to rename these enums to be something like `SPAN_BASED` and `VIEW_GRAVITY_BASED` to further clarify the purpose here.

Reply:
Comment is fine 👍. This PR has some links in the description anyway that could help trace the code that uses the VIEW_GRAVITY_BASED case.

>  I prefer to use an enum in cases like this...

Works for me! 👍